setup.py

```diff
@@ -4,12 +4,44 @@
 # This source code is licensed under the license found in the
 # LICENSE file in the root directory of this source tree.
 
+from pathlib import Path
+from typing import Iterable
+
+import pkg_resources
 from setuptools import find_packages, setup
 
+
+def _load_requirements(fname: str) -> Iterable[str]:
+    with open(Path(__file__).parent / fname) as fp_in:
+        for req in pkg_resources.parse_requirements(fp_in):
+            yield str(req)
+
+
+default_requirements = list(_load_requirements("requirements.txt"))
+dev_requirements = list(_load_requirements("dev_requirements.txt"))
+
 setup(
     name="seamless_communication",
-    version="0.1",
-    packages=find_packages(where="src"),
-    package_dir={"": "src"},
-    package_data={"": ["assets/cards/*.yaml"]},
+    version="1.0.0",
+    packages=find_packages(where="src") + ['m4t_scripts.finetune', 'm4t_scripts.predict'],
```

Comment:
Do we expect heavy code (that's not main entry) in finetune / predict or something importable? I was thinking we should move things that need to be imported to src/ all. But open to this change here.

Reply:
This is to expose CLI scripts as executables.